### ISSUE TITLE
Third SqlTypeModifier outputs instance name in createTable

### ISSUE THREAD

**Issue by jlandrum** [Kotlin/anko#373]
Adding 3 SqlTypeModifiers to a createTable call results in it outputting it's instance name instead of properly appending it's text to the query:
```kotlin
        db.createTable("tools", true,
                "mac" to TEXT + PRIMARY_KEY + UNIQUE,
                "type" to TEXT,
                "lastAdvertisement" to TEXT)
```

> Error Code : 1 (SQLITE_ERROR)
                                                 Caused By : SQL(query) error or missing database.
                                                 	(near "org": syntax error (code 1): , while compiling: CREATE TABLE IF NOT EXISTS `tools`(mac TEXT PRIMARY KEY org.jetbrains.anko.db.SqlTypeModifierImpl@82ed13f, type TEXT, lastAdvertisement TEXT);)

**Comment by jlandrum** [comment]
Turns out to have been a pretty simple oversight and a quick fix:
#374

**Comment by yanex** [comment]
Thank you for the contribution!

**Comment by ndac-todoroki** [comment]
I think this isn't fixed in the `1.10.1` source file (nor built through Gradle).
The master is fixed, but the changes are not included in the release somehow

**Comment by juanchosaravia** [comment]
This is true, just a small comment that the version is `0.10.0` that works and `0.10.1` doesn't have the fix.

**Comment by 4u7** [comment]
Fixed with a9fb4771c4ac8a211839bdb9174bf967595a36aa
